cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=None)
```

Comment:
The `description` field is defined as `serializers.CharField(max_length=None)`. In Django REST Framework, `CharField` should have a defined `max_length`. If you intend for this field to have no maximum length, consider using `serializers.CharField()` without specifying `max_length`, or use `serializers.CharField(max_length=some_value)` with a specific value.